scripts/runMainnet.sh

```diff
@@ -48,6 +48,7 @@ echo "MIN_RELAYER_FEE_PCT=0.0001" >> ${app_dir}/.env
 
 # Fee settings
 echo "PRIORITY_FEE_SCALER_1=0.8"  >> ${app_dir}/.env
+echo "RELAYER_GAS_PADDING=0"  >> ${app_dir}/.env
```

Comment:
Should we add similar configs to the remaining scripts to keep them consistent?

Reply:
You mean for Sepolia? Sepolia config varies a bit from the mainnet already and we could keep most of the configs default if no reason to change it. This value is tweaked especially for mainnet